.teamcity/Buildship/Promotion30/Project.kt

```diff
@@ -17,4 +18,8 @@ object Project : Project({
     buildType(Milestone)
 
     buildTypesOrder = arrayListOf(Release, Milestone, Snapshot)
+
+    params {
+        password("github.token", " credentialsJSON:3fcf7ce9-4dfe-477c-afc1-1da794a69214", label = "GitHub access token", display = ParameterDisplay.HIDDEN)
```

Comment:
What Github token is used here? I'd prompt for it when running the release promotion job and not hardcode it.

Reply:
Fixed.